tests/services/mixin_ipc_shm_service.py

```diff
@@ -91,11 +93,11 @@ def start_plasma_shm_mosec():
 
 if __name__ == "__main__":
     if len(sys.argv) < 2:
-        print("Please specify a shm to run: plasma or redis")
-        exit(1)
+        print("Please specify a shm storage service to run: plasma or redis")
+        sys.exit(1)
 
-    shm = sys.argv[1]
-    if shm == "plasma":
+    SERVICE = sys.argv[1]
```

Comment:
A minor issue: will `STORAGE` be better? `SERVICE` in our case more often refers to the mosec service itself?

Reply:
This is a service. If you choose "redis" it will run the redis-shm mosec service.